packages/web/components/related-assets/related-assets.tsx

```diff
@@ -0,0 +1,53 @@
+import Image from "next/image";
+
+import caretDown from "../../public/icons/caret-down.svg";
+
+export default function RelatedAssets() {
+  return (
+    <div className="flex flex-col gap-8 rounded-5xl border-[1px] border-osmoverse-800 bg-osmoverse-900 p-10">
```

Comment:
I suggest you to reduce the padding on mobile, so we'll avoid this effect:

<img width="340" alt="Screenshot 2023-09-25 alle 19 50 45" src="https://github.com/nabla-studio/osmosis-frontend/assets/17269969/55008ec2-64ec-46dd-9d91-8df881c2ae20">

Reply:
Let's try p-6 or it'll go on a new line on resolution such as "375px"